wifiphisher/pywifiphisher.py

```diff
@@ -960,12 +979,19 @@ def run():
 
     template.merge_context({'APs': APs_context})
 
-    template.merge_context({
-        'target_ap_channel': channel,
-        'target_ap_essid': essid,
-        'target_ap_bssid': ap_mac,
-        'target_ap_vendor': mac_matcher.get_vendor_name(ap_mac)
-    })
+    # added for custom AP (--apname)
+    if args.apname and args.channel:
+        template.merge_context({
+            'target_ap_channel': channel,
+            'target_ap_essid': essid,
+            'target_ap_bssid': ap_mac})
```

Comment:
I don't understand this branch here. Why are we not passing the `target_ap_vendor`?